.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.1.0
+    hooks:
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+  - repo: https://github.com/ansible/ansible-lint
+    rev: v4.0.1
+    hooks:
+      - id: ansible-lint
+        files: roles/.*\.(yaml|yml)$
```

Comment:
@Miouge1 My plan is to remove the roles/ include and lint everything but I want to do this gradually and this change is already very big. 

I am also sure that by the time everyone agrees on it it will not merge because it will need rebase and other fixes.

We may need to merge it as non-voting first (`|| true`) and make another small change later to make it voting.

Reply:
All right makes sense.

Reply:
This approach (start in a sub-folder and eventually widen scope) caused yaml files outside of `roles` to have tons of linting errors since introducing a yamllint CI step in #1556, see #4410